terraform-provider-constellation/internal/provider/cluster_resource_test.go

```diff
@@ -23,9 +23,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var providerVersion = semver.NewFromInt(2, 15, 0, "")
+const providerVersion string = "v0.0.0"
```

Comment:
Why do we need this here, when we overwrite it in the function scope?

Reply:
its not overwritten, see tests below

Reply:
```go
const providerVersion string = "v2.14.0"

func TestMicroserviceConstraint(t *testing.T) {
	providerVersion := semver.NewFromInt(2, 15, 0, "")
```

Do I misunderstand something here?

Reply:
If the const is just used in the function below, I think we should move it there.

Reply:
its used in several tests, keeping as constant therefore